### ISSUE TITLE
Carousel Block a11y improvements

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-ExhibitBuilder#147]
Carousel blocks need another look to fix some accessibility issues, to start:
- Pagination dots and next/prev arrows are lacking accessible names.
- Focus becomes lost at times when navigating via keyboard, especially backwards.

There are likely more issues to dig into here, but I just wanted to get an issue up while it was fresh.

**Comment by kimisgold** [comment]
The pagination dots and next/preview buttons have been converted from anchors to buttons with improved labeling. The pagination dots have been enlarged to 24x24px sizing and now use visible labels. These changes live in on the `a11y/carousel-block` branch.

I might need @matthewjmckinley's help on the keyboard navigation focus.

**Comment by matthewjmckinley** [comment]
I believe I fixed the issue with the navigation focus. jCarousel creates and destroys temporary clone/ghost slides when the carousel loops, and this was messing up the current focus cursor. My fix guards against that and makes sure the focus falls back somewhere that makes sense when the focus is on a temporary slide as its removed. LMK if it's still not working right.

Fixed in https://github.com/omeka/plugin-ExhibitBuilder/commit/97984a89d540aca0000d66b513fc7e91947d167e
